File: src/main/java/com/gtlugo/ampere/api/Grid.java
package com.gtlugo.ampere.api;

public class Grid {
  private double max_wattage;
  private double in_use_wattage;

  public void register() {

  }

  public boolean is_overloaded() {
    return max_wattage < in_use_wattage;
  }
}

git update-index --chmod=+x gradlew
git add .
git commit -m "Changing permission of gradlew"
git push

File: src/main/kotlin/com/gtlugo/ampere/api/Grid.kt
package com.gtlugo.ampere.api;

class Grid {
  private val maxWattage: Double
    get() {
      TODO()
    }

  private val inUseWattage: Double
    get() {
      TODO()
    }

  fun register() {
    TODO()
  }

  fun isOverloaded(): Boolean {
    return this.maxWattage < this.inUseWattage;
  }
}
